app/inpage/ts/inpage.ts

```diff
@@ -496,6 +496,17 @@ class InterceptorMessageListener {
 		return
 	}
 
+	private parseRpcError = (maybeErrorObject: unknown) => {
+		if (typeof maybeErrorObject !== 'object' || maybeErrorObject === null) return new EthereumJsonRpcError(METAMASK_ERROR_BLANKET_ERROR, 'Unexpected thrown value.', { maybeErrorObject } )
```

Comment:
Is building the object with named property `maybeErrorObject` intentional? This will return an object as `{ maybeErrorObject: null }` or something like `{ maybeErrorObject: 'An error message' }`

```suggestion
		if (typeof maybeErrorObject !== 'object' || maybeErrorObject === null) return new EthereumJsonRpcError(METAMASK_ERROR_BLANKET_ERROR, 'Unexpected thrown value.', { rawError: maybeErrorObject } )
```